code/game/machinery/doors/airlock_types.dm

```diff
@@ -179,6 +179,16 @@
 	var/mineral = "bananium"
 	doortype = /obj/structure/door_assembly/door_assembly_clown
 
+/obj/machinery/door/airlock/mime
+	name = "silencium airlock"
+	desc = "The exact opposite of a squeaky door."
+	icon = 'icons/obj/doors/airlocks/station/public.dmi'
+	color = "#87878715"
```

Comment:
Should the color var even be that? I was under the assumption it was the standard 6 digit RGB hex value